openstack/compute/v2/flavors/results.go

```diff
@@ -162,3 +162,43 @@ type FlavorAccess struct {
 	// TenantID is the unique ID of the tenant.
 	TenantID string `json:"tenant_id"`
 }
+
+func (r ExtraSpecsResult) Extract() (map[string]string, error) {
+	var s struct {
+		ExtraSpecs map[string]string `json:"extra_specs"`
+	}
+	err := r.ExtractInto(&s)
+	return s.ExtraSpecs, err
+}
+
+// ExtraSpecsResult contains the result of a call for (potentially) multiple
+// key-value pairs. Call its Extract method to interpret it as a
+// map[string]interface.
+type ExtraSpecsResult struct {
```

Comment:
if `ExtraSpecsResult` will only be used internally as the basis of other types, let's rename it to `extraSpecsResult`.

Reply:
Done